src/ploomber_engine/execute.py

```diff
@@ -74,9 +74,10 @@ def execute_notebook(
     cwd : str or Path, default='.'
         Working directory to use when executing the notebook
 
-    save_profiling_data : bool, default=False
+    save_profiling_data : bool or Path, default=False
         If True, saves profiling data generated from profile_memory and profile_runtime
-        (stores a ``.csv`` file in the same folder as ``output_path``)
+        (stores a ``.csv`` file in the same folder as ``output_path``).
```

Comment:
add some tests that invoke this function directly (as opposed to calling it from the CLI)

Reply:
add in https://github.com/ploomber/ploomber-engine/pull/78/commits/6372d47bee7f0a62afec5cac2de1b42f7c20c120